exporter.go

```diff
@@ -145,6 +179,16 @@ func main() {
 		return
 	}
 
+	// Only set process priority if a non-default and valid value has been set
+	if *processPriority != "normal" && priorityStringToInt[*processPriority] != 0 {
```

Comment:
Does `priorityStringToInt[*processPriority]` results into a panic, if `--process.priority=invalid` is given?

Reply:
If `processPriority` is ever `nil`, it'd result in a nil pointer de-reference. However a default value or user-supplied value is always set so that's not an issue.

If an invalid value such as `*processPriority="invalid"` is provided, it will result in the default `uint32` value of 0 and be ignored.